edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss

```diff
@@ -1741,3 +1741,13 @@ button {
     color: #3764f0;
   }
 }
+
+.btn.login-provider {
+  background: #126f9a;
```

Comment:
Is there a scss variable for: #126f9a? @andrey-canon

Reply:
no